bin/collector

```diff
@@ -1,6 +1,2 @@
 #!/usr/bin/env bash
-
-DEFAULT_TYPE=dev
-SERVICE_TYPE=${1:-$DEFAULT_TYPE}
-
-bin/sbt 'project zipkin-collector-service' "run -f zipkin-collector-service/config/collector-${SERVICE_TYPE}.scala"
+SERVICE_TYPE=${1:-dev} ./gradlew :zipkin-collector-service:run
```

Comment:
is it still possible to run https://github.com/openzipkin/zipkin/blob/master/zipkin-collector-service/config/collector-cassandra.scala ?